src/shared/components/case-viewer/case-viewer.component.html

```diff
@@ -38,16 +38,21 @@ <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-h
       </div>
     </div>
     <div *ngIf="hasEventSelector" class="column-one-half">
-      <ccd-event-trigger [isDisabled]="isTriggerButtonDisabled()" [triggers]="caseDetails.triggers"
-                         [triggerText]="triggerText" (onTriggerChange)="clearErrorsAndWarnings()"
-                         (onTriggerSubmit)="applyTrigger($event)"></ccd-event-trigger>
+      <ccd-event-trigger [isDisabled]="isTriggerButtonDisabled()"
+                         [triggers]="caseDetails.triggers"
```

Comment:
Again just formatting changes. So, please undo to avoid unnecessary history changes  [If possible]